scripts/imgtool.py

```diff
@@ -1,5 +1,6 @@
 #! /usr/bin/env python3
 #
+# Copyright 2018 Nordic Semiconductor ASA
```

Comment:
You can't really justify adding a copyright statement to a file you aren't otherwise modifying.

Reply:
Yes, that was an oversight. Will remove.